runtime/src/handler.rs

```diff
@@ -67,6 +67,8 @@ pub trait Handler {
 	fn exists(&self, address: H160) -> bool;
 	/// Check whether an address has already been deleted.
 	fn deleted(&self, address: H160) -> bool;
+	/// Check whether an address has already been created.
+	fn created(&self, address: H160) -> bool;
```

Comment:
Looks like this function is never used from the `Handler`. `StackExecutor` uses it in its internal implementations, but that doesn't mean it needs to be part of the `Handler` interface.

Reply:
Make sense. Fixed.